package io.limberapp.backend.module.users.exception.user

import io.limberapp.common.exception.notFound.EntityNotFound

class UserNotFound : EntityNotFound("User")
